datascience/tables.py

```diff
@@ -1202,23 +1259,26 @@ def sample(self, k=None, with_replacement=True, weights=None):
         """Returns a new table where k rows are randomly sampled from the
         original table.
 
-        Kwargs:
-            k (int or None): If None (default), all the rows in the table are
-                sampled. If an integer, k rows from the original table are
-                sampled.
+        Args:
+            ``k`` -- specifies the number of rows (``int``) to be sampled from
+               the table. Default is k equal to number of rows in the table.
+
+            ``with_replacement`` -- (``bool``) By default TRUE; Samples ``k``
+                rows with replacement from table, else samples ``k`` rows
+                without replacement.
 
-            with_replacement (bool): If True (default), samples the rows with
-                replacement. If False, samples the rows without replacement.
+            ``weights`` -- Array specifying probability the ith row of the
+                table is sampled. If None, by default, ``weights`` is the
+                uniform distribution on [1, ... , n], n = number of rows.
```

Comment:
I don't think this is more clear than before, which states:

    If None (default), samples the rows using a uniform random distribution.